pyproject.toml

```diff
@@ -54,7 +54,11 @@ benchmark = [
 
 
 [project.entry-points.sklearn_engines]
-kmeans = "sklearn_numba_dpex.cluster.kmeans:KMeansEngine"
+kmeans = "sklearn_numba_dpex.kmeans.engine:KMeansSklearnEngine"
+
+[project.entry-points.pytest11]
```

Comment:
What does `pytest11` stands for, here?

Reply:
I had the same question. Is this because this is version 11 of the plugin API of pytest?